schemas/tsp/booking-options-list/request.json

```diff
@@ -82,6 +82,11 @@
     },
     "legGeometry": {
       "$ref": "../../core/components/units-geo.json#/definitions/polyline"
+    },
+    "extraOptions": {
```

Comment:
I agree. I think our endpoint should not fail if any extra properties are passed. That said it's also good practice to have some control which parameters are used so that we are not just using random parameters

Reply:
@medikoo @konker Backend guys wanted to rename it from tspProductIds that was suggested in slack chat to `extraOptions`.

Reply:
Also changed `additionalProperties` to `true`